home/service/search.py

```diff
@@ -71,7 +71,7 @@ def _build_entity_types(_, entity_types: list[str]) -> tuple[ResultType]:
     def _get_search_results(self, page: str, items_per_page: int) -> SearchResponse:
         form_data = self.form_data
 
-        query = form_data.get("query", "")
+        query = form_data.get("query", "").replace("_", " ")
```

Comment:
Maybe add a comment here pointing at the datahub behaviour that introduced the bug? Then we can revert this if/when they fix it